processor/remotetapprocessor/README.md

```diff
@@ -11,8 +11,8 @@
 [alpha]: https://github.com/open-telemetry/opentelemetry-collector#alpha
 [contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
 <!-- end autogenerated section -->
-The WebSocket processor, which can be positioned anywhere in a pipeline, allows
-data to pass through to the next component. Simultaneously, it makes a portion
+The Remote App processor a.k.a. WebSocket processor can be positioned anywhere in a pipeline.
```

Comment:
This is not a Remote App processor, but indeed a Remote Tap, as in the ability to tap into the flow.

Reply:
@atoulme Yep, silly mistake on my part. I've pushed a fix. I didn't mean to open the PR yet and was still checking it.